source/design-and-content/index.html.md.erb

```diff
@@ -3,4 +3,24 @@ title: Design and content
 weight: 12
 ---
 
-# Design and content
\ No newline at end of file
+# Design and content
+
+## Designers on the team
+
+### Chris
```

Comment:
I think this should probs just be in alphabetical order tbh!

Reply:
Cool, I can get it changed, and for some reason I had to do 2 separate requests :(

Reply:
Resolving this conversation as Ciandelle addressed it in another commit.